circle.yml

```diff
@@ -17,9 +17,12 @@ machine:
 dependencies:
   cache_directories:
     - "~/cache"
+    - "~/.npm"
   override:
     - sudo apt-get -q update; sudo apt-get install -q -y libgif-dev libsasl2-dev
     - ./scripts/install_cmake.sh
+    # The real npm cache is in ~/.npm, but CircleCI tries to cache node_modules, breaking upgrades
+    - rm -rf ./node_modules
```

Comment:
It's weird that this is necessary.  Wouldn't this also imply that deleting node_modules is needed for developers as wel